src/site/content/en/blog/async-clipboard/index.md

```diff
@@ -0,0 +1,412 @@
+alt: Clipboard with shopping list
+---
+
+Over the past few years, browsers have used
+`[document.execCommand()](https://developers.google.com/web/updates/2015/04/cut-and-copy-commands)`
+for clipboard interactions. Though widely supported, this method of cutting and
+pasting came at a cost: clipboard access was synchronous, and could only read
+and write to the DOM.
+
+That's fine for small bits of text, but there are many cases where blocking the
+page for clipboard transfer is a poor experience. Time consuming sanitization or
+image decoding might be needed before content can be safely pasted. The browser
+may need to load or inline linked resources from a pasted document. That would
+block the page while waiting on the disk or network. Imagine adding permissions
+into the mix, requiring that the browser block the page while requesting
+clipboard access. At the same time, the permissions put in place around
+`document.execCommand()` for clipboard interaction are loosely defined and vary
+between browsers.
+
+The
+[Async Clipboard API](https://www.w3.org/TR/clipboard-apis/#async-clipboard-api)
+addresses these issues, providing a well-defined permissions model that doesn't
+block the page. Safari recently announced  [support for it in version
+13.1](https://webkit.org/blog/10855/). With that, major browsers have a basic
+level of support in place. As of this writing, Firefox only supports text; and
+image support is limited to PNGs in some browsers. If you're interested in using
+the API,
+[consult a browser support table](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard#Browser_compatibility)
+before proceeding.
+
+This article explains how to write both text and images to, and read them from
+the clipboard. This article does not cover [Feature Policy for the
+clipboard](https://www.chromestatus.com/features/5767075295395840) which landed
+in Chrome&nbsp;85.
+
+{% Aside %}
+The Async Clipboard API is limited to handling text and images. Chrome&nbsp;84
+introduces an experimental feature that allows the clipboard to handle any
+arbitrary data type. 
+{% endAside %}
+
+## Copy: writing data to the clipboard
+
+### writeText()
+
+To copy text to the clipboard call `writeText()`. Since this API is
+asynchronous, the `writeText()` function returns a Promise that resolves or
+rejects depending on whether the passed text is copied successfully:
+
+```js
+async function copyPageUrl() {
+  try {
+    await navigator.clipboard.writeText(location.href);
+    console.log('Page URL copied to clipboard');
+  } catch (err) {
+    console.error('Failed to copy: ', err);
+  }
+}
+```
+
+### write()
+
+Actually, `writeText()` is just a convenience method for the generic `write()`
+method, which also lets you copy images to the clipboard. Like `writeText()`, it
+is asynchronous and returns a Promise.
+
+To write an image to the clipboard, you need the image as a
+`[blob](https://developer.mozilla.org/en-US/docs/Web/API/blob)`. One way to do
```

Comment:
Looks like lots of links have the formating error

![image](https://user-images.githubusercontent.com/4713486/88985780-1cde4200-d286-11ea-8ce6-d0273aea44fa.png)

Reply:
Probably the Markdown conversion extension…